playbooks/roles/ses-osh/tasks/main.yml

```diff
@@ -11,6 +11,15 @@
   with_items:
     - "{{ hostvars[groups['ses_nodes'][0]].ceph_files.results }}"
 
+- name: Get key values for ses-config.yml
+  set_fact:
+    "ceph_{{ item | regex_replace('[\\.\\-]', '_') }}_key":
+       "{{lookup('ini', 'key section={{ item }} file={{ socok8s_workspace }}/ceph.{{ item }}.keyring') }}"
+  with_items:
```

Comment:
Shouldn't this be based on the ceph_files.results too, to avoid having an item list that's out of date, and ensuring everything is properly fetched?